doc/model.tex

```diff
@@ -306,13 +306,13 @@ \section{Model: mango }
       \textbf{vodml-id: EpochPositionErrors.properMotion} \newline
       \textbf{type: \hyperref[sect:error.PropertyError]{mango:error.PropertyError}} \newline
       \textbf{multiplicity: 1} \newline
-      Position error; can be an ellipse, a correlation matrix or a covariance matrix.
+      Proper motion error; can be an ellipse or a matrix.
```

Comment:
I think this should be ellipse or Symmetrical2D, the same as for Position error.  The same text appears in diffs for multiple files, I don't know which if any is the original source.

Reply:
The process of building the document is rather though. (see utils/README.md)